src/celeritas/phys/PhysicsData.hh

```diff
@@ -239,6 +239,10 @@ struct PhysicsParamsScalars
     Energy lowest_electron_energy{};  //!< Lowest e-/e+ kinetic energy
     real_type linear_loss_limit{};  //!< For scaled range calculation
     real_type fixed_step_limiter{};  //!< Global charged step size limit [len]
+    real_type lambda_limit{};  //!< lambda limit
+    real_type range_fact{};  //!< range factor for e-/e+ (0.2 for muon/h)
+    real_type safety_fact{};  //!< safety factor
```

Comment:
Can you put these in a separate MSC section and note that the documentation for them is in `\c PhysicsParams`? Also is there a reason we abbreviate "factor" as "fact"? It doesn't save that much space...

Reply:
Sure. The name would be a question for @whokion :) I'm happy to add the `or`.